docs/tools/network_ip_addresses.mdx

```diff
@@ -8,17 +8,60 @@ The details of your specific integration should have been included with your rob
 
 This chart is for reference, to help you decide what address to assign sensors or computers that you are adding to a robot.
 
-| Device                                      | Start IP or range | End IP of range |
-| :------------------------------------------ | :---------------- | :-------------- |
-| Main Robot Computer                         | 192.168.131.1     | -               |
-| Main Robot Microcontroller                  | 192.168.131.2     | -               |
-| Additional Computers and Control Equipment  | 192.168.131.3     | 192.168.131.9   |
-| Camera, Vision Sensor, Illumination         | 192.168.131.10    | 192.168.131.19  |
-| LiDAR                                       | 192.168.131.20    | 192.168.131.29  |
-| GPS                                         | 192.168.131.30    | 192.168.131.34  |
-| Sonar                                       | 192.168.131.35    | 192.168.131.39  |
-| Arm                                         | 192.168.131.40    | 192.168.131.44  |
-| Gripper                                     | 192.168.131.45    | 192.168.131.49  |
-| Network Hardware                            | 192.168.131.50    | 192.168.131.69  |
-| Unassigned (for future static IP addresses) | 192.168.131.70    | 192.168.131.99  |
-| DHCP pool of addresses                      | 192.168.131.100   | 192.168.131.254 |
+| Device Family     | Device Description      | Static IP Address/Range  | Comments                                     |
+| :---------------- | :---------------------- | :----------------------- | :------------------------------------------- |
+| Computers & MCUs  | Primary PC              | 192.168.131.1            | The `roscore` process always runs on this PC |
```

Comment:
IMO "Computer" > "PC", since it's really a headless embedded computer anyway, but I don't know how deeply the PC-centric nomenclature is ingrained in other parts of the literature.

Reply:
That's a relatively easy change. I think across all of our documentation we use "PC" and "Computer" interchangeably, but given that the main computer could be a mini ITX, NUC, Jetson, etc... it's probably good to start moving away from using "PC".